frontend/pages/app/sign-in.vue

```diff
@@ -1,9 +1,34 @@
 <script lang="ts" setup>
+import { ref } from 'vue';
+
+const email = ref('');
+const password = ref('');
 definePageMeta({
   layout: "auth-layout",
 });
-const signIn = () => {
-  useRouter().push('/app')
+const signIn = async () => {
+  
+  const response = await fetch('http://localhost:9001/api/v1/auth', {
```

Comment:
Eventually this needs to be moved into a pinia store. .vue files should be doing HTTP requests through the store.